tsconfig.json

```diff
@@ -26,7 +26,7 @@
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
 
     /* Modules */
-    "module": "ESNext",
+    "module": "NodeNext",
```

Comment:
I've changed this, because `ESNext` is not a valid value here, when the `moduleResolution` is set to `NodeNext`. VSCode warns about it too:

![image](https://github.com/tolgee/tolgee-cli/assets/31937175/69dddf6d-b942-4ccd-b172-1c3a25bd9ddd)

The build output doesn't change, so it's not a braking change or anything.